src/plugins/beeline/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @author Ryadnov Andrey <me@ryadnov.ru>
+ */
+
+import * as main from "../cft/main";
+
+/**
+ * @type {string}
```

Comment:
jsdoc annotations for simple stuff like this look messy

imagine comments like
`"value goes here" // it's a string`